content/en/docs/rules/fspath.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Accessing File System Paths in Falco Rules
+description: How fs.path.* fields work
+linktitle: Accessing File System Paths
+weight: 100
+---
+
+## Introduction
+
+This section explains how the fields `fs.path.*` work and when they can be used.
+
+### Motivation
+
+A variety of different syscalls take file system paths as arguments. However, there is little consistency in the fields that can access those file system paths. Depending on the syscall, the file system path might be available in `evt.rawarg.path`, `evt.rawarg.pathname`, `evt.rawarg.name`, `fd.name`, etc. Depending on the specific libs event identifier (this generally changes over time based on the falco version, or based on the libc version), the field might differ. And for some system calls, the file system path is actually in the enter event instead of the exit event. This makes it very difficult to write simple Falco rules that act on file system paths, because the field must depend on the syscall as well as the event direction.
+
+To help address this inconsistency, in Falco version XXX we added a new set of fields that normalize file system paths across a variety of syscalls.
+
+### `fs.path.*` fields
+
+The following fields are available for any syscall that operates on a file system path:
```

Comment:
We had a discussion about this before, perhaps state that this does not apply to when spawning a process that uses an executable on disk, aka proc.exepath is kept solely in the proc field class for xyz reasons ...